src/goto-instrument/dump_c.cpp

```diff
@@ -1521,22 +1521,48 @@ void dump_ct::cleanup_type(typet &type)
     !type.get(ID_tag).empty());
 }
 
+static expr2c_configurationt expr2c_configuration()
+{
+  static expr2c_configurationt configuration{
```

Comment:
IMHO would be better written as a series of assignments, then you don't need the comments.